src/lib.rs

```diff
@@ -188,10 +189,9 @@ impl<'a> Clay<'a> {
         unsafe { Clay_Hovered() }
     }
 
-    // TODO: Uncomment once `clay.h` adds the declaration of `Clay_PointerOver`
```

Comment:
Remove this comment.

Reply:
It is removed

Reply:
:facepalm: I haven't had enough coffee today it seems